cla_public/apps/checker/fields.py

```diff
@@ -161,10 +161,22 @@ def __init__(self, label=None, validators=None, min_val=0, max_val=9999999999, *
         self.min_val = min_val
         self.max_val = max_val
 
+    def convert_amount_to_utf8(self, amount):
+        # Covert pounds side of the input into UTF-8 format
+        try:
+            amount = amount.decode("utf-8")
+        except UnicodeEncodeError:
+            # Input is already in UTF-8 format
+            pass
+
+        return amount
+
     def process_formdata(self, valuelist):
         if valuelist:
```

Comment:
We could reduce cyclomatic complexity by switching from
```
if valuelist:
    do stuff
<implicitly returns None otherwise>
```
to
```
if not valuelist:
    return
do stuff
```

Reply:
Since I have still kept the helper function and modified like discussed below, there's no need to do this anymore